### DISCUSSION TITLE
Automatic backup when save changes

### DISCUSSION THREAD

**Discussion by VerNOX21** [mtkennerly/ludusavi#381] [Ideas]
It would be incredibly convenient if the program tracked the save file, and in the case when a save occurred in the game and the file date changed, the program would make a new backup. Then you wouldn't have to regularly minimize the game and make a copy manually. Is it possible to implement this in principle?

**Comment by mtkennerly** [comment]
Hi! I considered something like this before (https://github.com/mtkennerly/ludusavi/issues/211), but it's pretty difficult to get right. There are some alternatives you can try that get pretty close: https://github.com/mtkennerly/ludusavi/issues/211#issuecomment-2162011527 . Personally, I would recommend setting up a [scheduled backup](https://github.com/mtkennerly/ludusavi/blob/master/docs/help/backup-automation.md) (e.g., nightly).
